docs/configuration_file.md

```diff
@@ -101,11 +101,19 @@ Some options are also mandatory.
 
 * `pulp_secret` (*optional*, `string`) — name of [kubernetes secret](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md) to use for pulp plugin
 
-* `pdc_secret` (*optional*, `string`) - name of kubernetes secret to use for plugins communicating with PDC
+* `smtp_host` (*optional*, `string`) - SMTP server host, e.g. `smtp.mycompany.com`
 
-* `pdc_url` (*optional*, `string`) - URL of PDC server instance, e.g. `https://pdc.mycompany.com`
+* `smtp_from` (*optional*, `string`) - Address to send notifications from, e.g. `user@mycompany.com`
 
-* `smtp_uri` (*optional*, `string`) - URI of SMTP server, e.g. `smtp.mycompany.com`
+* `smtp_additional_addresses` (*optional*, `string`) - Additional addresses to include in notifications, e.g. `user1@mycompany.com, user2@mycompany.com`
```

Comment:
Worth including "comma-separated" in there (and for other '_addresses') to make it explicit?